src/blocks.zig

```diff
@@ -386,8 +386,8 @@ pub const Block = packed struct { // MARK: Block
 		return _touchFunction[self.typ];
 	}
 
-	pub fn entityDataClass(self: Block) ?*EntityDataClass {
-		return _entityDataClass[self.typ];
+	pub fn blockEntity(self: Block) ?*BlockEntityType {
```

Comment:
Did you intentionally omit `...Type` suffix here?

Reply:
yes